wallet-as-a-service/wallet-experience-apis/post-v2-batch-floor-prices.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "POST Batch Floor Prices"
+api: "POST https://api.bitski.com/v2/floor-prices/batch?supportedChainIds={chainIds}"
+description: "Get floor prices for a wallet on a chain"
+---
+
+### Query
```

Comment:
We've been calling this `Parameters` in other files.

Reply:
sounds reasonable